application-xpoll-test/application-xpoll-test-docker/src/test/it/com/xwiki/xpoll/test/ui/PollIT.java

```diff
@@ -90,10 +90,18 @@ class PollIT
 
     private static final String POLL_PUBLICITY_PRIVATE_LABEL = "Private";
 
-    private static final String EMPTY_PAGE_VOTE_WARNING = "You are not allowed to access this content.";
+    /**
+     * XWIKI-22433 introduced a new way of displaying a warning using Velocity macro. When XWiki version is greater than
```

Comment:
You should also add the issue title. It is important to have at least this info, to not rely on the issue tracker to get everything. This way you don't force the person which reads your comment to search it on the issue tracker, as only the title can be enough information sometimes.

```suggestion
     * XWIKI-22433: Provide consistency in info / warning / error class usages in templates - this task introduced a new way of displaying a warning using Velocity macro. When XWiki version is greater than
```